python/ray/autoscaler/autoscaler.py

```diff
@@ -379,29 +385,29 @@ def _update(self):
             if node_ip in last_used and last_used[node_ip] < horizon and \
                     len(nodes) - num_terminated > target_workers:
                 num_terminated += 1
-                print("StandardAutoscaler: Terminating idle node: "
-                      "{}".format(node_id))
+                logger.info("StandardAutoscaler: Terminating idle node: "
+                            "{}".format(node_id))
                 self.provider.terminate_node(node_id)
             elif not self.launch_config_ok(node_id):
                 num_terminated += 1
-                print("StandardAutoscaler: Terminating outdated node: "
-                      "{}".format(node_id))
+                logger.info("StandardAutoscaler: Terminating outdated node: "
+                            "{}".format(node_id))
                 self.provider.terminate_node(node_id)
         if num_terminated > 0:
             nodes = self.workers()
-            print(self.debug_string(nodes))
+            logger.info(self.debug_string(nodes))
 
         # Terminate nodes if there are too many
         num_terminated = 0
         while len(nodes) > self.config["max_workers"]:
             num_terminated += 1
-            print("StandardAutoscaler: Terminating unneeded node: "
-                  "{}".format(nodes[-1]))
+            logger.info("StandardAutoscaler: Terminating unneeded node: "
```

Comment:
Do we need the "StandardAutoscaler" prefix now that we have logger? Presumably it already prefixes the log w/something useful?

Reply:
I didn't format the logger output (so right now there's no prefix)

Reply:
I agree with @ericl here. I think this is then a separate discussion what should Ray use for a default formatting string.

Reply:
Sure - I think the issue is more broadly scoped than this PR and will leave the status quo.